tests/unit/components/frost-text-test.js

```diff
@@ -23,5 +32,124 @@ describeComponent(
     it('includes className frost-text', function () {
       expect(component.classNames).to.include('frost-text')
     })
+
+    it('sets default property values correctly', function () {
+      expect(
+        component.get('align'),
+        'align: left'
+      ).to.be.eql('left')
+
+      expect(
+        component.get('autocapitalize'),
+        'autocapitalize: none'
+      ).to.be.eql('none')
+
+      expect(
+        component.get('autocorrect'),
+        'autocorrect: off'
+      ).to.be.eql('off')
+
+      expect(
+        component.get('autofocus'),
+        'autofocus: false'
+      ).to.be.false
+
+      expect(
+        component.get('isClearEnabled'),
+        'isClearEnabled: false'
+      ).to.be.false
+
+      expect(
+        component.get('isClearVisible'),
+        'isClearVisible: false'
+      ).to.be.false
+
+      expect(
+        component.get('isHookEmbedded'),
+        'isHookEmbedded: false'
+      ).to.be.false
+
+      expect(
+        component.get('readonly'),
+        'readonly: false'
+      ).to.be.false
+
+      expect(
+        component.get('required'),
+        'required: false'
+      ).to.be.false
+
+      expect(
+        component.get('selectionDirection'),
+        'selectionDirection: none'
+      ).to.be.eql('none')
+
+      expect(
+        component.get('spellcheck'),
+        'spellcheck: false'
+      ).to.be.false
+
+      expect(
+        component.get('tabindex'),
+        'tabindex: 0'
+      ).to.be.eql('0')
```

Comment:
You can change this to `0` once you update the default value to not be a string.